java-rmi/test/com/baeldung/rmi/JavaRMITest.java

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.rmi;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.rmi.NotBoundException;
+import java.rmi.RemoteException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+
+import org.junit.Test;
+
+public class JavaRMITest {
+	
+	@Test
+	public void WhenRunServer_ServerIsStarted() {
```

Comment:
Site standard is givenXXX_whenYYY_thenZZZ or whenYYY_thenZZZ, so this would be called: whenRunServer_thenServerStarts(). Note the camelCase as well as the additional "then"

Reply:
Will do this.